cmd/thor/flags.go

```diff
@@ -59,6 +59,10 @@ var (
 		Name:  "api-allow-custom-tracer",
 		Usage: "allow custom JS tracer to be used tracer API",
 	}
+	apiLogsEnabledFlag = cli.BoolFlag{
```

Comment:
Should it be `cli.StringFlag` and the user specifies the log file location?

Reply:
I think we can probably add that flag if this feature takes traction. But I imagine it's easier to have this boolean flag with a known directory (`instaceDir`) instead of a three option flag (DefaultDir, Specify `null` to disable or Specify `path` to enable feature and set the log path). How does that sound to you ?